modules/mobile/spec/controllers/profile_base_controller_spec.rb

```diff
@@ -11,7 +11,10 @@
       allow(Rails.logger).to receive(:warn)
 
       if type == :sis
+        user = sis_user
         @new_headers = sis_headers
+        # prevent fingerprint mismatch that raises a warning and complicates tests
+        request.remote_ip = user.fingerprint
```

Comment:
This is kinda stupid. This seems to happen in perhaps all SIS request specs, but it doesn't cause issues because the authenticate method just logs and warning then resets the user fingerprint. But the codes fail otherwise because they're testing logging. We added these specs because the feature was specific to logging, but logging tests are a pain for exactly this reason. There's probably a better way of ignoring other logs and narrowing in on the log in question, but I'm not sure what it. Line 11 above already catches one instance. Possible I could've fixed this problem by just adding `.twice` but I didn't try that.